mobile/library/cc/engine_builder.h

```diff
@@ -170,6 +170,15 @@ class EngineBuilder {
   EngineBuilder& enforceTrustChainVerification(bool trust_chain_verification_on);
   EngineBuilder& setUpstreamTlsSni(std::string sni);
   EngineBuilder& enablePlatformCertificatesValidation(bool platform_certificates_validation_on);
+  EngineBuilder& setUseQuicPlatformPacketWriter(bool use_quic_platform_packet_writer);
```

Comment:
Is this an experimental API, which will eventually be replaced, or do we think this will be used indefinitely.